src/components/Toc/TocSections/TocSections.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import {block} from '../../utils/cn';
+import {TocItem} from '../TocItem';
+import type {TocItem as TocItemType} from '../types';
+
+import './TocSections.scss';
+
+const b = block('toc-sections');
```

Comment:
```suggestion
const b = block('toc');
```

Let's keep it part of the `toc` block